src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimReissueService.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.events.claim.CitizenClaimCreatedEvent;
+import uk.gov.hmcts.cmc.claimstore.events.claim.PostClaimOrchestrationHandler;
+import uk.gov.hmcts.cmc.claimstore.events.solicitor.RepresentedClaimCreatedEvent;
+import uk.gov.hmcts.cmc.claimstore.idam.models.User;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import static uk.gov.hmcts.cmc.domain.models.ClaimState.CREATE;
+
+@Service
+public class ClaimReissueService {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private final ClaimService claimService;
+    private final UserService userService;
+    private final PostClaimOrchestrationHandler postClaimOrchestrationHandler;
+
+    public ClaimReissueService(
```

Comment:
How did we autowire all the services? who calls the constructor?

Reply:
added the autowired